src/app/zap-templates/templates/app/gen_config.zapt

```diff
@@ -4,7 +4,7 @@
 #pragma once
 
 // User options for plugin Binding Table Library
-#define EMBER_BINDING_TABLE_SIZE 10
+#define EMBER_BINDING_TABLE_SIZE 64
```

Comment:
we only want to update the size on the casting app

Reply:
Removed the change from the common zap template and only retained the change in the tv-casting-app specific file.